packages/hub/db/migrations/20210809113449561_merchant-infos.js

```diff
@@ -0,0 +1,19 @@
+const TABLE = 'merchant_infos';
+
+exports.up = (pgm) => {
+  pgm.createTable(TABLE, {
+    id: { type: 'uuid', primaryKey: true },
+    merchant_name: { type: 'string', notNull: true },
+    merchant_slug: { type: 'string', notNull: true },
+    color: { type: 'string', notNull: true },
+    text_color: { type: 'string', notNull: true },
+    owner_address: { type: 'string', notNull: true },
+    created_at: { type: 'timestamp', notNull: true, default: pgm.func('current_timestamp') },
+  });
+
+  pgm.createIndex(TABLE, 'merchant_slug', { unique: true });
```

Comment:
Unique index assures there will be no collisions here. 

However, I just noticed this will allow inserting same strings in different case (upper/lower) which will be a problem. I'm thinking about forcing lowercase before persisting. Thoughts?

Reply:
+1 to unique index. I suggest considering using the `citext` module. I believe it is supported on AWS Postgres RDS.

Reply:
Thanks, didn't know this exists! 

I'm a bit reluctant to include new DB extensions unless absolutely necessary (there could be installation issues in various environments, and it's usually something extra to take care of during db provisioning). In this case it can be a simple `toLowercase` when inserting the payload and when filtering. Let me know if you think there's a better solution, but I'm pretty happy with the one I mentioned (will implement this a bit later).